### ISSUE TITLE
suggestion

### ISSUE THREAD

**Issue by Zuhyy** [SebSemos/F.M.R.P-Freshly-Modded-#1]
can you add alex's mobs to your freshly modded resource pack so that it works with fresh animations? thanks (for 1.21 fabric)

**Comment by SebSemos** [comment]
Yes, this one been on my to do list, but It's gonna take some time :(

**Comment by digi2303** [comment]
First there aren't any Alex's Mobs port for Fabric 1.21 iirc, second he can't make it because Citadel is not supported by EMF
